contracts/token/ERC20/StandardBurnableToken.sol

```diff
@@ -8,7 +8,7 @@ import "./StandardToken.sol";
  * @title Standard Burnable Token
  * @dev Adds burnFrom method to ERC20 implementations
  */
-contract StandardBurnableToken is BurnableToken, StandardToken {
+contract StandardBurnableToken is StandardToken, BurnableToken {
```

Comment:
Since `BurnableToken` now inherits from `StandardToken` (and therefore has the `allowed` state variable`, shouldn't we merge this contract with `BurnableToken`?